.drone.yml

```diff
@@ -61,6 +59,7 @@ steps:
       - echo $version > /mnt/botamusique/version
       - sed -i "s/version = 'git'/version = '$version'/" mumbleBot.py
       - curl --silent "https://api.github.com/repos/azlux/botamusique/releases/latest" | jq -r '.body' | pandoc --from gfm --to html - --output - > /mnt/botamusique/changelog
+      - (cd lang && ./sync_translation.py)
       - rm -rf .git*
```

Comment:
with the `cd` you are in the bad folder to the next step.

Reply:
No. If I cd in `()`, it won't affect the next line